packages/utils/src/snark-artifacts/config.ts

```diff
@@ -1,9 +1,13 @@
-import { Artifact, Proof, SnarkArtifacts, Version } from "../types"
+import { Artifact, Proof, Version } from "../types"
 
-const ARTIFACTS_BASE_URL = "https://unpkg.com/@zk-kit"
+const getBaseUrls = (proof: Proof, version: Version) => [
```

Comment:
my initial idea was to shuffle this randomly, but as https://github.com/privacy-scaling-explorations/snark-artifacts/pull/23 told us how to rank them by download speed, we should follow that order